aip/0216.md

```diff
@@ -69,6 +69,22 @@ progression through a lifecycle.
 
 ## Additional Guidance
 
+### Default value
+
+The zero value of each state enum **should** follow a consistent convention:
+
+```proto
+enum State {
+  // The default value. This value is used if the state is omitted.
+  STATE_UNSPECIFIED = 0;
+
+  // Other values...
+}
+```
+
+Resources **should not** provide an unspecified state to users, and this value
+**should not** actually be used.
```

Comment:
We used to say that it was okay for the 0 value to be a sort of "simple default" where that was genuinely useful. So is this a deliberate change in policy?

Reply:
I think that makes sense in an AIP about enums generally, but for _states_, I am fairly certain that an explicit unspecified is always warranted. Thoughts?

Reply:
Ah, right. I hadn't spotted that this was just about state enums. That's fine.

Reply:
Yes, for States the zero-value wouldn't make a lot of sense (since states should not be set using an Update or Create method but are modified only by other actions).